vector/src/main/res/xml/vector_settings_labs.xml

```diff
@@ -53,14 +53,8 @@
 
     <im.vector.app.core.preference.VectorSwitchPreference
         android:defaultValue="false"
-        android:key="SETTINGS_LABS_ENABLE_POLLS"
-        android:title="@string/labs_enable_polls" />
```

Comment:
Can you add a comment above the string resource `labs_enable_polls` like `<!-- TODO Remove -->` please?

Reply:
Done, I was thinking to ask you about this.